Camilla cluedo step 1 done

camillalof opened the pull request:
step 1 done

camillalof committed:
Camilla cluedo step 1 done

camillalof committed:
Camilla cluedo itteration 2 and 3 done

camillalof committed:
Camilla cluedo itteration 4 done

camillalof committed:
Camilla cluedo itteration 4 plus a little styleing

ericaglimsholt commented on code/style.css:
```diff
@@ -10,13 +10,13 @@ p {
   display: flex;
   justify-content: space-around;
   align-items: center;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```
I can't see that you have included this font to the project. This means that I can't see the font. If you want to include it you can read up on it here: https://www.pagecloud.com/blog/how-to-add-custom-fonts-to-any-website

ericaglimsholt commented on code/style.css:
```diff
@@ -47,6 +54,12 @@ p {
 }
 
+h1 {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```
Same goes for here - I can't see this font

ericaglimsholt reviewed (approved):
Good work on your Cluedo assignment! I hope it was fun for you. I think you did great, and you did follow the requirements of the assignment. One thing that I would love to see being more developed are the default state (when you enter the page). Here you got an "error" in your image because the `src` is undefined until you click on the killer. A suggestion here is to add a default image to the stack of cards (maybe the back of the card would look good?). I've also added a comment about the fonts. It's really nice that you have used different fonts - that's always nice to see. But for others to see it you have to include it to the project. I've added a comment for you down below with a link where you can read up on it more! 

🌟

camillalof commented:
Thank you! I will look at it. I saw the problem with the image but couldn’t figure out how to make it default. Can you please explain some more on how to do that? :)

/Camilla

Skickat från min iPhone

29 okt. 2019 kl. 15:27 skrev Erica Glimsholt <notifications@github.com<mailto:notifications@github.com>>:


@ericaglimsholt approved this pull request.

Good work on your Cluedo assignment! I hope it was fun for you. I think you did great, and you did follow the requirements of the assignment. One thing that I would love to see being more developed are the default state (when you enter the page). Here you got an "error" in your image because the src is undefined until you click on the killer. A suggestion here is to add a default image to the stack of cards (maybe the back of the card would look good?). I've also added a comment about the fonts. It's really nice that you have used different fonts - that's always nice to see. But for others to see it you have to include it to the project. I've added a comment for you down below with a link where you can read up on it more!

🌟

________________________________

In code/style.css<https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102409>:

>    display: flex;
   justify-content: space-around;
   align-items: center;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;


I can't see that you have included this font to the project. This means that I can't see the font. If you want to include it you can read up on it here: https://www.pagecloud.com/blog/how-to-add-custom-fonts-to-any-website

________________________________

In code/style.css<https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102691>:

> @@ -47,6 +54,12 @@ p {
   border-radius: 16px;
 }

+h1 {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;


Same goes for here - I can't see this font

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Technigo/project-cluedo/pull/4?email_source=notifications&email_token=AKNPJJDO66KZWO6CXU7SG6LQRBB5LA5CNFSM4JEDGAWKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJSASFA#pullrequestreview-308545812>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKNPJJBEWFVWB5GSO363WBLQRBB5LANCNFSM4JEDGAWA>.

ericaglimsholt commented:
> Thank you! I will look at it. I saw the problem with the image but couldn’t figure out how to make it default. Can you please explain some more on how to do that? :) /Camilla Skickat från min iPhone 29 okt. 2019 kl. 15:27 skrev Erica Glimsholt <notifications@github.com<mailto:notifications@github.com>>: @ericaglimsholt approved this pull request. Good work on your Cluedo assignment! I hope it was fun for you. I think you did great, and you did follow the requirements of the assignment. One thing that I would love to see being more developed are the default state (when you enter the page). Here you got an "error" in your image because the src is undefined until you click on the killer. A suggestion here is to add a default image to the stack of cards (maybe the back of the card would look good?). I've also added a comment about the fonts. It's really nice that you have used different fonts - that's always nice to see. But for others to see it you have to include it to the project. I've added a comment for you down below with a link where you can read up on it more! 🌟
> ________________________________ In code/style.css<[#4 (comment)](https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102409)>:
> display: flex;
> justify-content: space-around; align-items: center; + font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif; I can't see that you have included this font to the project. This means that I can't see the font. If you want to include it you can read up on it here: https://www.pagecloud.com/blog/how-to-add-custom-fonts-to-any-website
> ________________________________ In code/style.css<[#4 (comment)](https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102691)>:
> @@ -47,6 +54,12 @@ p {
> border-radius: 16px; } +h1 { + font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif; Same goes for here - I can't see this font — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub<#4?email_source=notifications&email_token=AKNPJJDO66KZWO6CXU7SG6LQRBB5LA5CNFSM4JEDGAWKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJSASFA#pullrequestreview-308545812>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKNPJJBEWFVWB5GSO363WBLQRBB5LANCNFSM4JEDGAWA>.

Of course! Here's two small, easy ways to it:
1 - Delete the `src` property on the `<img>`
2 - Add a source to the `src` property on `<img>`, like `assets/plum.png`

If you go with option one or two, it will update ones you click on the card and call your function

camillalof commented:
I tried both ways, and it still is a frame of the image before pushing the card? Is there another way to change the picture?

/Camilla

Skickat från min iPhone

29 okt. 2019 kl. 16:14 skrev Erica Glimsholt <notifications@github.com<mailto:notifications@github.com>>:


Thank you! I will look at it. I saw the problem with the image but couldn’t figure out how to make it default. Can you please explain some more on how to do that? :) /Camilla Skickat från min iPhone 29 okt. 2019 kl. 15:27 skrev Erica Glimsholt <notifications@github.com<mailto:notifications@github.com>mailto:notifications@github.com>: @ericaglimsholt<https://github.com/ericaglimsholt> approved this pull request. Good work on your Cluedo assignment! I hope it was fun for you. I think you did great, and you did follow the requirements of the assignment. One thing that I would love to see being more developed are the default state (when you enter the page). Here you got an "error" in your image because the src is undefined until you click on the killer. A suggestion here is to add a default image to the stack of cards (maybe the back of the card would look good?). I've also added a comment about the fonts. It's really nice that you have used different fonts - that's always nice to see. But for others to see it you have to include it to the project. I've added a comment for you down below with a link where you can read up on it more! 🌟
________________________________ In code/style.css<#4 (comment)<https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102409>>:
display: flex;
justify-content: space-around; align-items: center; + font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif; I can't see that you have included this font to the project. This means that I can't see the font. If you want to include it you can read up on it here: https://www.pagecloud.com/blog/how-to-add-custom-fonts-to-any-website
________________________________ In code/style.css<#4 (comment)<https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102691>>:
@@ -47,6 +54,12 @@ p {
border-radius: 16px; } +h1 { + font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif; Same goes for here - I can't see this font — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub<#4<https://github.com/Technigo/project-cluedo/pull/4>?email_source=notifications&email_token=AKNPJJDO66KZWO6CXU7SG6LQRBB5LA5CNFSM4JEDGAWKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJSASFA#pullrequestreview-308545812>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/AKNPJJBEWFVWB5GSO363WBLQRBB5LANCNFSM4JEDGAWA.

Of course! Here's two small, easy ways to it:
1 - Delete the src property on the <img>
2 - Add a source to the src property on <img>, like assets/plum.png

If you go with option one or two, it will update ones you click on the card and call your function

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Technigo/project-cluedo/pull/4?email_source=notifications&email_token=AKNPJJFZL5ALGYEPLL7ZUC3QRBHOBA5CNFSM4JEDGAWKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECQ4HJQ#issuecomment-547472294>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKNPJJHM3DZGCF3S5RCE2O3QRBHOBANCNFSM4JEDGAWA>.

ericaglimsholt commented:
Can you make a commit when you have added the image, so I can see what's wrong?

camillalof committed:
Camilla cluedo again

camillalof commented:
I made the commit. All pictures started to be wrong now. Thank you for helping!

/Camilla

Skickat från min iPhone

29 okt. 2019 kl. 16:14 skrev Erica Glimsholt <notifications@github.com<mailto:notifications@github.com>>:


Thank you! I will look at it. I saw the problem with the image but couldn’t figure out how to make it default. Can you please explain some more on how to do that? :) /Camilla Skickat från min iPhone 29 okt. 2019 kl. 15:27 skrev Erica Glimsholt <notifications@github.com<mailto:notifications@github.com>mailto:notifications@github.com>: @ericaglimsholt<https://github.com/ericaglimsholt> approved this pull request. Good work on your Cluedo assignment! I hope it was fun for you. I think you did great, and you did follow the requirements of the assignment. One thing that I would love to see being more developed are the default state (when you enter the page). Here you got an "error" in your image because the src is undefined until you click on the killer. A suggestion here is to add a default image to the stack of cards (maybe the back of the card would look good?). I've also added a comment about the fonts. It's really nice that you have used different fonts - that's always nice to see. But for others to see it you have to include it to the project. I've added a comment for you down below with a link where you can read up on it more! 🌟
________________________________ In code/style.css<#4 (comment)<https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102409>>:
display: flex;
justify-content: space-around; align-items: center; + font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif; I can't see that you have included this font to the project. This means that I can't see the font. If you want to include it you can read up on it here: https://www.pagecloud.com/blog/how-to-add-custom-fonts-to-any-website
________________________________ In code/style.css<#4 (comment)<https://github.com/Technigo/project-cluedo/pull/4#discussion_r340102691>>:
@@ -47,6 +54,12 @@ p {
border-radius: 16px; } +h1 { + font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif; Same goes for here - I can't see this font — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub<#4<https://github.com/Technigo/project-cluedo/pull/4>?email_source=notifications&email_token=AKNPJJDO66KZWO6CXU7SG6LQRBB5LA5CNFSM4JEDGAWKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJSASFA#pullrequestreview-308545812>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/AKNPJJBEWFVWB5GSO363WBLQRBB5LANCNFSM4JEDGAWA.

Of course! Here's two small, easy ways to it:
1 - Delete the src property on the <img>
2 - Add a source to the src property on <img>, like assets/plum.png

If you go with option one or two, it will update ones you click on the card and call your function

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Technigo/project-cluedo/pull/4?email_source=notifications&email_token=AKNPJJFZL5ALGYEPLL7ZUC3QRBHOBA5CNFSM4JEDGAWKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECQ4HJQ#issuecomment-547472294>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKNPJJHM3DZGCF3S5RCE2O3QRBHOBANCNFSM4JEDGAWA>.

camillalof committed:
finished with syiling

camillalof committed:
finished

camillalof committed:
Update README.md

camillalof committed:
finished2

camillalof committed:
no photos

camillalof committed:
finished without imgages

camillalof committed:
finished

camillalof committed:
Update README.md
